lib/ProductOpener/Tags.pm

```diff
@@ -215,6 +216,37 @@ sub has_tag($$$) {
 	return $return;
 }
 
+# Determine if a tag is a child of another tag (ot the same tag)
+# assume tags are already canonicalized
+sub is_a($$$) {
+	
+	my $tagtype = shift;
+	my $child = shift;
+	my $parent = shift;
+	
+	#$log->debug("is_a", { tagtype => $tagtype, child => $child, parent => $parent }) if $log->is_debug();
```

Comment:
Commented log statments here and following lines.

Reply:
I still don't understand how to set the log level in product opener, what I put in log.conf seems to be ignored. How do I get debug messages in Apache's error log? (or in another file)